fimpy/pipeline/alignment.py

```diff
@@ -8,6 +8,8 @@
     shift_stack,
     sobel_stack,
 )
+from skimage.feature import register_translation
```

Comment:
I thought we switched to https://scikit-image.org/docs/stable/api/skimage.registration.html?highlight=phase_cross_correlation#phase-cross-correlation with the new scikit-image

Reply:
True, I missed this. I switched the functions.